lib/cli.rb

```diff
@@ -21,6 +21,7 @@ class Cli < Thor
   class_option :version, :type => :boolean, :desc => "Show version"
   class_option :offline, :type => :boolean, :desc => "Work offline"
   class_option :manifest_dir, :type => :string, :desc => "Manifest directory"
+  class_option :templates, :type => :string, :desc => "Switch templates"
```

Comment:
This should not be necessary. The option is an option to the view method, not to the class.